### PR TITLE
Lectia 6

### PR DESCRIPTION
**Pull request by GR-Viktor**

### CODE REVIEW

File: scss/components/_transitions.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  &::after {
+    content: '';
+    display: block;
+    padding-top: 20%;
+  }
+
+  &:hover {
+    .transition-child {
+      left: calc(100% - 50px);
```
Comment (alexcracea) [inline scss/components/_transitions.scss]: Please apply this suggestion
```suggestion
      left: calc(100% - 60px);
```
your transition child has grown to 60 in width + height, then this transition should also reflect that

File: scss/components/_transitions.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  left: 0;
+}
+
+.transition-child--1 {
+  transition-duration: 2s;
+  transition-timing-function: cubic-bezier(0.35, 0.27, 0.62, 0.93);
+}
+
+.transition-child--2 {
+  padding-top: 60px;
```
Comment (alexcracea) [inline scss/components/_transitions.scss:33]: suggestion: no need to use padding for all these transition childs, they are all position: absolute already, you can simply use the `top` here

File: scss/layout/_ratio-box.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+}
+
+.ratio-box1 {
```
Comment (alexcracea) [inline scss/layout/_ratio-box.scss]: Please remove ratio-box1 because its duplicate

File: scss/layout/_ratio-box.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
+@media 
+(-webkit-min-device-pixel-ratio: 2), 
+(min-resolution: 192dpi) { 
+
+}
```
Comment (alexcracea) [inline scss/layout/_ratio-box.scss]: Please remove this empty code block

File: scss/components/_gradient.scss [hunk 5]
```diff
@@ -1,6 +1,18 @@
...
+    padding-top: 75%;
+    display: block;
+  }
+
+  > * {
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
```
Comment (alexcracea) [inline scss/components/_gradient.scss:15]: please remove this duplicate code from gradient box. just use the `ratio-box` class on those divs

File: scss/components/_filters.scss [hunk 6]
```diff
@@ -1,10 +1,23 @@
...
     display: block;
+  }
+
+  > * {
+    position: absolute;
+    top: 0;
+    left: 0;
     width: 100%;
+    height: 100%;
+    object-fit: cover;
```
Comment (alexcracea) [inline scss/components/_filters.scss:15]: please remove the duplicate from filters box, just use the `ratio-box` class on those divs

File: scss/base/_helpers.scss [hunk 7]
```diff
@@ -0,0 +1,40 @@
+@mixin fix-inline-blocks($fontsize: 16px) {
+  font-size: 0;
+
+  > * {
+    display: inline-block;
+    font-size: $fontsize;
+  }
+}
+
+.ratio-box1 {
```
Comment (alexcracea) [inline scss/base/_helpers.scss]: please remove `ratio-box1` duplicates. All ratio box classes should come only in a single copy located in the `_ratio-box.scss` file

File: index.html [hunk 8]
```diff
@@ -129,5 +130,148 @@ <h2 class="section__title">Gradient</h2>
       </div>
     </div>
   </div>
+
+  <div class="section" id="transform">
```
Comment (alexcracea) [inline index.html:134]: All these sections of Gradient, Transform, Transition, Animation are currently outside the 
```
<main>
  <content class="content-area">
    (all sections should be here)
  </content>
</main>
```

**Review by alexcracea — CHANGES_REQUESTED** [review]

File: scss/components/_transitions.scss (hunk 2 above)
Comment (GR-Viktor) [reply scss/components/_transitions.scss:33]: Cum spunea Olguta - eu oleaca "tupesc"...
Nu inteleg ce aveti in vedere...

**Pull request merged**